src/components/Search/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { get } from "lodash";
+import { push } from "react-router-redux";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { Input } from "react-materialize";
+import Button from "../Button";
+import SelectInput from "../SelectInput";
+import cities from "../../config/cities";
+
+// used for calendar component
+import "materialize-css/dist/css/materialize.min.css";
+import "materialize-css/dist/js/materialize.min.js";
+
+class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      source: "",
+      destination: "",
+      date: "" // this is stored as a 13 digit epoch time
+    };
+  }
+
+  handleChange = (e, input) => {
+    this.setState({
+      [input]: get(e, "value", "")
+    });
+  };
+
+  onFind = () => {
+    // TODO: Check if all fields are complete and mabye push /rides/<params> ?
+    this.props.changePage("/rides");
+    // TODO: need to dispatch a fetch to the api here to get list of rides
+  };
+
+  render() {
+    return (
+      <div className="search">
+        <h3>Find a ride</h3>
+        <h5>
+          Currently we only support UCLA as an origin. Check back later for more
+          locations!
+        </h5>
+
+        <SelectInput
```

Comment:
Maybe change the landing page to a login/register page. Once logged in, the user can register a car, post a listing, and view/request available rides.

Reply:
yea i like that. we could either do that or keep the landing page as is and just let users try to find rides but redirect them to a login/register page if they're not logged in